app/javascript/application.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import { Provider } from "react-redux";
+import store from "../redux/store";
+import App from "../components/App";
+
+document.addEventListener("DOMContentLoaded", () => {
+  ReactDOM.render(
```

Comment:
Good job making the required changes asked by the previous reviewer, Unfortunately, The app still does not work as expected, please consider the following changes to make the app run as expected:

- [ ] The app still gives me a black screen when I run the dev server, and when I tried to precompile assets before that it gave me the errors in the video below. So please fix that and make sure that the app is functioning as expected and showing random greeting messages.

https://github.com/Gardimy/Set_up_hello_project_webpack/assets/68796232/fc7e1455-d94d-4453-99d0-8765cdb49b11

[Hint] All the errors while precompiling came from:
1. not installing `webpack` and `webpack-cli` packages.
2. The paths of the `import` in some `.js` files.